js/waiting.js

```diff
@@ -0,0 +1,34 @@
+(function() {
+  var s = (new Date()).getTime();
+  var tmp = new moment.duration(moment(new Date(launchDate).getTime())-s);
```

Comment:
Not crazy about this variable name 😄

Reply:
addressed in commit: c1b63e5b775889446974e4509b87fa5a2dd2b7cb

Reply:
👍 much more readable, thanks!